tests/integration/test_dbapi_integration.py

```diff
@@ -859,3 +860,78 @@ def test_info_uri(trino_connection):
     cur.fetchall()
     assert cur.info_uri is not None
     assert cur._query.query_id in cur.info_uri
+
+
+def test_client_tags_single_tag(run_trino):
+    _, host, port = run_trino
+
+    client_tags = ["foo"]
+    trino_connection = trino.dbapi.Connection(
+        host=host,
+        port=port,
+        user="test",
+        source="test",
+        max_attempts=1,
```

Comment:
Remove unnecessary arguments, `source` and `max_attempts` unrelate to client tag test.

Reply:
Reminder.

Reply:
Missed that one!